run_kraken.py

```diff
@@ -210,6 +212,49 @@ def time_scenarios(scenarios_list, config):
                 publish_kraken_status(config, not_reset)
 
 
+def litmus_scenarios(scenarios_list, config, litmus_namespaces):
+    # Loop to run the scenarios starts here
+    for l_scenario in scenarios_list:
+        try:
+            for item in l_scenario:
+                runcommand.invoke("kubectl apply -f %s" % item)
+                if "http" in item:
+                    f = requests.get(item)
+                    yaml_item = list(yaml.safe_load_all(f.content))[0]
+                    logging.info("yaml item "  + str(yaml_item))
+                else:
+                    with open(item, "r") as f:
+                        logging.info("opened yaml" + str(item))
+                        yaml_item = list(yaml.safe_load_all(f))[0]
+
+                if yaml_item['kind'] == "ChaosEngine":
+                    engine_name = yaml_item['metadata']['name']
+                    namespace = yaml_item['metadata']['namespace']
+                    litmus_namespaces.append(namespace)
+                    experiment_names = yaml_item['spec']['experiments']
+                    for expr in experiment_names:
+                        expr_name = expr['name']
+                        experiment_result = common_litmus.check_experiment(engine_name,
+                                                                           expr_name,
+                                                                           namespace)
+                        if experiment_result:
+                            logging.info("Scenario: %s has been successfully injected!"
+                                         % item)
+                        else:
+                            logging.info("Scenario: %s was not successfully injected!"
+                                         % item)
+            for item in l_scenario:
+                logging.info('item ' + str(item))
+                runcommand.invoke("kubectl delete -f %s" % item)
+            cerberus_integration(config)
+            logging.info("Waiting for the specified duration: %s" % wait_duration)
+            time.sleep(wait_duration)
```

Comment:
Why not wait for a condition rather than wait for a fix amount of time? Otherwise how do you know if the condition has been met after the wait_duration?

Reply:
This is just a wait time between scenarios to give the cluster a break

Reply:
what do you mean by a break (cpu utilization, network IO)? Is there any way you can synchronize this wait to a signal (deployment of a pod completed, or something else)?
Waiting is best to link it to a condition, otherwise it just delays the execution and can render flaky tests results.

Reply:
If you take a look at the configuration (tunings-> wait_duration (https://github.com/paigerube14/kraken/blob/320c9531bf7b4b0a3df8202a4290abbf12a1a208/config/config.yaml#L26)) we want to wait in-between executions of separate scenarios. This should have no effect on the test results